tools/vagrant/README.md

```diff
@@ -176,46 +186,46 @@ cd ${OPENWHISK_HOME}
 ./gradlew distDocker
 ```
 
-## Safe Re-deploy (after VM restart)
+## Using docker-runc
+Only for experimental use:
+To use docker-runc the docker-runc CLI use in the invoker needs to match the version on the docker engine host.
+Get the version of the docker engine like the following:
+```
+$ docker version | grep Version
+Version:	18.03.0-ce
+```
+Update the Invoker Dockerfile to download a newer version of the docker-cli
```

Comment:
i think this sentence is implying 
1. compare the docker-runc version obtained on the local system against the docker-runc configured for the invoker
2. if the versions are different, only then do you need to update the invoker dockerfile to point to the matching docker download